### ISSUE TITLE
implement MLP

### ISSUE THREAD

**Issue by markroxor** [jellAIfish/omega#2]
Implement multi-layer perceptron model in neural networks, starting with stochastic gradient descent algorithm.

**Comment by markroxor** [comment]
WIP in #1
